### PR TITLE
Feature/page content per lang

### PR DESCRIPTION
**Pull request by jskowronski39**
This will:
* Add separate partial views for page sections with language-specific, complex formatting.
* Fix some code formatting issues etc

### CODE REVIEW

**Comment by veteran29** [comment]
Very nice! Could you do same treatment for "join us" page?

File: templates/home/join_us.html.twig [hunk 1]
```diff
@@ -11,53 +11,53 @@
...
+                                <ul id="join-steps">
+                                    <li>
+                                        Zacznij od zapoznania się z <a href="{{ url.statute }}">regulaminem</a>
+                                    </li>
+                                    <li>
+                                        Dołącz do naszego <a href="{{ url.discord }}">Discorda</a> i złóż podanie na kanale #rekrutacja
+                                    </li>
+                                    <li>
+                                        Gdy już otrzymasz rangę rekruta będziesz mógł zalogować się na stronie i dołączyć do naszego
+                                        {%- if is_granted('ROLE_USER') -%}
```
Comment (veteran29) [inline templates/home/join_us.html.twig]: This white space should not be stripped as this results in no space between words.
```suggestion
                                        {%- if is_granted('ROLE_USER') %}
```

Comment (jskowronski39) [reply templates/home/join_us.html.twig]: Fixed

**Comment by jskowronski39** [comment]
> 
> 
> Very nice! Could you do same treatment for "join us" page?

Done

**Pull request merged**